src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -145,7 +145,7 @@
     /** Information about the EC2 instance (if any) this worker is running on. */
     EC2Info ec2info;
 
-    /** TODO what's this number? */
+    /** The time the last high priority request was processed, used to check if the machine should be shut down. */
```

Comment:
I think we should always mention units on any number, even when it seems obvious.